Source/santa-driver/SantaDecisionManager.h

```diff
@@ -89,6 +89,9 @@ class SantaDecisionManager : public OSObject {
 
   ///  Clears the cache.
   void ClearCache();
+  
+  /// Checks for vnode_id in the cache. Returns santa_action_t for the vnode_id.
+  santa_action_t CheckCache(uint64_t vnode_id);
```

Comment:
This is a little confusing as there's already a method called CacheCheck() for invalidating entries, that method should be renamed RemoveFromCache. It probably isn't really necessary to have this ChechCache function at all, instead making GetFromCache public and moving the vnode->string conversion into SantaDriverClient::check_cache()